src/fibonacciBases.js

```diff
@@ -0,0 +1,89 @@
+function convertToFib() {
+  let number = document.getElementById('number').value
+  let result = []
+  let fibonacci = []
+
+//calculate fibonacci value
+  function fibonacciValue(resultArr) {
+    let value = 0
+    for(let i = 0; i < fibonacci.length; i++) {
+      value += ( resultArr[i] * fibonacci[i] )
+    } return value
+  }
+  // why doesnt this work?
```

Comment:
Again, is this for a tutorial. Awful lot of comments. You may also want to consider putting these comments in a gist.